### PR TITLE
Periodic job for Serverless on GCP

### PR DESCRIPTION
**Pull request by mgencur**
* runs once a week

Fixes https://issues.redhat.com/browse/SRVCOM-669

### CODE REVIEW

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master__4.3-gcp.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+base_images:
+  base:
+    name: '4.3'
+    namespace: ocp
+    tag: base
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+tests:
+- as: e2e-aws-ocp-43-gcp-continuous
```
Comment (markusthoemmes) [inline ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master__4.3-gcp.yaml]: Should this not have "aws" in it?

Comment (mgencur) [reply ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master__4.3-gcp.yaml]: Oops.

**Comment by mgencur** [comment]
Pushed a fix, also for vSphere and Azure jobs.

**Pull request merged**